_posts/2020-02-13-better-sleep.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Better Sleep - Timings
+date: 2020-01-13 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I used to go to sleep at 1am on a regular basis and wake up at 8:30. That would leave me with an average of 7 hours of sleep. Not awful, but surely not good. Not counting the days when I had to wake up at 7 for one or the other reason.
+
+During the past year I always went to bed at (or before) 21:30.
+
+That would not mean that I would fall asleep right away at 21:30 obviously, sometimes I won't even fall asleep by 22, but most surely I will be before 23. The average is more closer to 22 though.
+
+I have an alarm for 7 since I'm trying to wake up without rushing.[^1] Usually though, I find myself waking up without needing it at around 6:30 (after more than 8 hours of sleep).
+
+Some nights when I fall asleep earlier I manage to get close to 9 hours which is fine every once in a while, but too much sleep is not great either.
+
+An interesting change with this routine is that during the weekdays I have the same sleeping schedule as during the weekends. There was a time when I woke up at 10 or even 11 on Saturdays and Sundays. I'm glad it is no more.
+
+I realize this is hard if you're used to go to sleep at midnight as I used do, but you do not need to start so drastically as I did. You can add 30 minutes to your sleep time and that's going to do wonders already. If you're not ready to make the big jump, try making smaller ones.
+
+TLDR: Go to sleep earlier.
+
+[When I started the rough draft of this article I though it would end up being a single short piece on what were the changes I went through to sleep better. There weren't really that many I thought. It ended up being way longer than expected so I've split it into multiple entries.]
+
+[^1]: I used to have this problem when I woke up at 8:30 and was not able to do everything I wanted to before having to go to work.
+[^2]: That does not mean that I do not like what I do at my job. I love my job, but it is still a job.
```

Comment:
If you were back in Italy working your 1st job you would have left by 7 and came at midnight - commute, mandatory lunch break, being overworked - > not everyone wants to work all day and then just sleep once they are home... spending time for yourself + family AND going to sleep early is impossible if you work.

Reply:
Yeah true that